src/main/resources/templates/addFacility.html

```diff
@@ -14,25 +14,25 @@
 
     <ul class="nav">
         <li><a href="index.html">Search</a></li>
-        <li><a href="AddFacility.html">Add Faculty</a></li>
+        <li><a href="addFacility.html">Add Faculty</a></li>
     </ul>
 
     <div class="findBox">
 
         <p>Add a new Facility</p>
 
-        <form action="#" th:action="@{/saveFacility}" th:object="${facility}" method="GET">
+        <form action="#" th:action="@{/saveFacility}" th:object="${facility}" method="POST">
         <label for="facilityName"></label>
-        <input type="text" id="facilityName" name="facility" placeholder="Facility Name" th:field="facilityName">
```

Comment:
Your th:field expression was incorrect and caused the following runtime exception.

Expression "facilityName" is not valid: only variable expressions ${...} or selection expressions *{...} are allowed in Spring field bindings (template: "addfacility" - line 26, col 90)

I would recommend changing this, so you do not encounter this exception.

Reply:
Good.